CHANGELOG.md

```diff
@@ -1,5 +1,150 @@
 # Change Log
 
+## v5.12.0
+
+## :tada: Features
+
+-   [feature] Configure format from file (#4155)
+-   [feature] Make cli expect array for rules dir (#3788)
+-   [feature] Set js rules to all valid active rules (#3641)
+-   [improvement] Add tests for typescript@2.9 and typescript@3.0 (#4053)
+-   [new-fixer][type-literal-delimiter](https://palantir.github.io/tslint/rules/type-literal-delimiter/) (#3964)
```

Comment:
Nit: formatting is off for type-literal-delimiter (missing backticks). Here and lower down too.